ground/src/main/java/com/google/android/ground/ui/map/gms/mog/MogTile.kt

```diff
@@ -39,7 +39,7 @@ class MogTile(val metadata: MogTileMetadata, val data: ByteArray) {
   private fun app0Segment() =
     APP0_MARKER +
       APP0_MIN_LEN.toByteArray() +
-      JFIF_IDENTIFIER.toNulTerminatedByteArray() +
+      JFIF_IDENTIFIER.toNullTerminatedByteArray() +
```

Comment:
Spelling "NUL" (one "L") was intentional here:

null = special value in Kotlin/Java
NUL = \u0000 (!=null)

I may have misused this in at least one other place, so should be changed to NUL there as well.